### PR TITLE
[feat] Implement the settings tab of the experiment page

### PR DESCRIPTION
**Pull request by VkoHov**
Implement the settings tab of the experiment page

### CODE REVIEW

File: aim/web/ui/src/modules/core/api/experimentsApi/index.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ async function getExperimentById(id: string): Promise<IExperimentData> {
  * @returns {Promise<status: string, id: string>}
  */
 async function updateExperimentById(
-  reqBody: { name?: string; archived?: boolean },
+  reqBody: { name?: string; description?: string; archived?: boolean },
```
Comment (rubenaprikyan) [inline aim/web/ui/src/modules/core/api/experimentsApi/index.ts:55]: Please create type for this to use inside components as well

**Review by KaroMourad — APPROVED** [review]
lgtm

File: aim/web/ui/src/pages/Experiment/ExperimentStore.ts [hunk 2]
```diff
@@ -17,13 +22,43 @@ function experimentEngine() {
...
 
+  function updateExperiment(name: string, description: string) {
+    const experimentData = experimentState.getState().data;
+    updateExperimentById(
+      { name, description, archived: experimentData?.archived },
+      experimentData?.id || '',
+    )
+      .then((res) => {
+        experimentState.setState((prev: any) => ({
+          ...prev,
```
Comment (rubenaprikyan) [inline aim/web/ui/src/pages/Experiment/ExperimentStore.ts:33]: doesn't this work without `...prev`? zustand merges states instead of replacing

Comment (VkoHov) [reply aim/web/ui/src/pages/Experiment/ExperimentStore.ts:33]: You were right. I have removed it.

**Review by rubenaprikyan — APPROVED** [review]
Lgtm!

**Pull request merged**